scripts/bump.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-current_version=$(tomlq -r '.workspace.dependencies.wind_tunnel_core.version' Cargo.toml)
+current_version=$(tq -r '.workspace.dependencies.wind_tunnel_core.version' Cargo.toml)
```

Comment:
Same binary provided by https://search.nixos.org/packages?channel=25.05&show=tomlq&query=tomlq but I don't find this clearer, is `tomlq` such a long name? Yes if you're typing it but in a script perhaps it was fine?